RadixWallet/Core/SharedModels/P2P/ConnectorExtension/P2P+ConnectorExtension+Request.swift

```diff
@@ -74,20 +76,15 @@ extension P2P.ConnectorExtension.Request {
 				let ledgerDevice: P2P.LedgerHardwareWallet.LedgerDevice
 				let compiledTransactionIntent: HexCodable
 				let displayHash: Bool
-				let mode: String
+				let mode: String = "summary"
+			}
 
-				init(
-					signers: [P2P.LedgerHardwareWallet.KeyParameters],
-					ledgerDevice: P2P.LedgerHardwareWallet.LedgerDevice,
-					compiledTransactionIntent: HexCodable,
-					displayHash: Bool
-				) {
-					self.signers = signers
-					self.ledgerDevice = ledgerDevice
-					self.compiledTransactionIntent = compiledTransactionIntent
-					self.mode = "summary"
-					self.displayHash = displayHash
-				}
+			struct SignPreAuthorization: Sendable, Hashable, Encodable {
```

Comment:
Should be called SignSubintentHash, maybe worth taking time and to migrate the Ledger models into Sargon